addons/mass_mailing/tests/test_mailing_ui.py

```diff
@@ -18,9 +18,12 @@ def test_01_mass_mailing_editor_tour(self):
         # The tour created and saved an email. The edited version should be
         # saved in body_arch, and its transpiled version (see convert_inline)
         # for email client compatibility should be saved in body_html. This
-        # ensures both fields have different values.
-        self.assertEqual(mail.body_arch, '<p><br></p>')
-        self.assertEqual(mail.body_html, '<p style="margin:0px 0 14px 0;box-sizing:border-box;"><br></p>')
+        # ensures both fields have different values (the mailing body should
+        # have been converted to a table in body_html).
+        self.assertTrue(mail.body_arch.find('data-snippet="s_title"'))
```

Comment:
Hello @Zinston 

Would you mind having a quick look at this modification?
It seems to still do the appropriate check, but I prefer to be sure.

Thanks!

Reply:
Hi @awa-odoo ! Looks good to me, this is gonna be more robust btw, thanks for the change ;-)